test/e2e/serial/workload_placement_test.go

```diff
@@ -71,10 +70,15 @@ var _ = Describe("[serial][disruptive][scheduler] workload placement", func() {
 		err = fxt.Client.List(context.TODO(), &nrtList)
 		Expect(err).ToNot(HaveOccurred())
 
-		tmPolicy := nrtv1alpha1.SingleNUMANodeContainerLevel
-		nrts = e2enrt.FilterTopologyManagerPolicy(nrtList.Items, tmPolicy)
+		// we're ok with any TM policy as long as the updater can handle it,
+		// we use this as proxy for "there is valid NRT data for at least X nodes
+		policies := []nrtv1alpha1.TopologyManagerPolicy{
```

Comment:
why we need this change in this split?

Reply:
still here, I don't think we need this change. Could you please elaborate?

Reply:
we don't need it in this split, after a second thought better to have it in a separate patch. I'll update it. thanks.